samples/sensor/accel_stream/sample.yaml

```diff
@@ -0,0 +1,7 @@
+sample:
+  name: Accel Streaming Sample
+tests:
+  sample.sensor.accel_stream:
+    harness: sensor
```

Comment:
shouldn't this be `harness: console` and have a basic sanity check of the output using a `single_line` or `multi_line` regex?

applies to other samples as well.

Reply:
dropped the sample for now (can readd post 3.7)